pkg/utils/utils.go

```diff
@@ -443,3 +444,15 @@ func Cast(v any, to any) {
 	toVal = toVal.Elem()
 	toVal.Set(reflect.ValueOf(v))
 }
+
+func GetDaemonAuth() eosclient.Authorization {
+	return eosclient.Authorization{Role: eosclient.Role{UID: "2", GID: "2"}}
```

Comment:
Is it ok to hardcode or should we make this configurable?

Reply:
I think it is ok to hardcode (maybe in a `const`), as this is hardcoded in EOS, and all this logic is EOS-specific.